src/MetricSelect/MetricSelectScene.tsx

```diff
@@ -576,14 +576,46 @@ export class MetricSelectScene extends SceneObjectBase<MetricSelectSceneState> i
                 <>
                   <div className={styles.displayOptionTooltip}>
                     Filter by
-                    <IconButton
-                      name={'info-circle'}
-                      size="sm"
-                      variant={'secondary'}
-                      tooltip="This switch enables filtering by OTel resources for OTel native data sources."
-                    />
+                    <Tooltip
+                      content={
+                        <div>
+                          <p>The OTel experience is deprecated in Grafana Metrics Drilldown.</p>
+                          <p>
+                            Please use the following docs to promote your OTel resource attributes to metrics with{' '}
```

Comment:
```suggestion
                            Please use the following docs to promote your OTel resource attributes as metric labels with{' '}
```

Should we say "promote as metric labels"?